File: assignment 10 - C# practice problems/js/13.js
//Write a JavaScript program to check whether a given 
//stringInputing contains equal number of p's and t's.
function checkIFStringContainsEqualPAndTDriverFunction() {
    let stringInput = prompt("Enter a string");

    if (stringInput != null) {
        stringInput = stringInput.split("");

        let pCount = 0;
        let tCount = 0;

        stringInput.forEach(element => {

            switch (element) {
                case 'p':
                    ++pCount;
                    break;
                case 't':
                    ++tCount;
                    break;
            }
        });

        if (pCount === tCount) {
            alert("p and t count is equal");
        } else {
            alert("p and t count is not equal");
        }
    }
}
checkIFStringContainsEqualPAndTDriverFunction()

File: assignment 10 - C# practice problems/13.cs
//Write a c# program to check whether a given 
//string contains equal number of p's and t's.
using System;
namespace MyNameSpace
{
    class Program13
    {
        public static void CheckIFStringContainsEqualPAndTDriverFunction()
        {

            Console.WriteLine("Enter a string: ");
            string inputSrring = Console.ReadLine();

            var inpurArray = inputSrring.ToCharArray();
            var pCount = 0;
            var tCount = 0;

            foreach (var charecter in inpurArray)
            {
                if (charecter == 'p' || charecter == 'P')
                {
                    pCount += 1;
                }
                else if (charecter == 't' || charecter == 'T')
                {
                    tCount += 1;
                }
            }

            if (pCount == tCount)
            {
                Console.WriteLine("p and t count is equal");
            }
            else
            {
                Console.WriteLine("p and t count is not equal");
            }

        }
    }
}
